.github/ISSUE_TEMPLATE/late_release_exception.yml

```diff
@@ -0,0 +1,75 @@
+# This issue template is for use in opening issues that document late release exception requests.
+# 
+# Bracketed text is a placeholder; replace the text with the requested information and remove the brackets before submitting the issue. Also, remove this comment before submitting the issue.
+
+name: "IdentityModel Release Exception Request"
+description: Request an exception to release a feature or add any changes after the release candidate deadline, excludes fixes for preview regressions.
+title: "[Late Release Exception]: "
+labels: "Late Release Exception"
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Brief description of the feature.
+    validations:
+      required: true
+  - type: input
+    id: release-date
+    attributes:
+      label: Date
+      description: Which release date are you targeting?
+    validations:
+      required: true
+  - type: dropdown
+    id: version-bump
+    attributes:
+      label: Type of version bump
+      description: Indicate if this feature would incur a major, minor, or patch version bump. 
+      options:
+        - label: "Major"
+        - label: "Minor"
+        - label: "Patch"
+        - label: "Not sure"
+    validations:
+      required: true
+  - type: checkboxes
+    id: other-libraries
+    attributes:
+      label: Other libraries
+      description: Does this feature release rely on the release of any other ID4S libraries?
```

Comment:
Make required?

Reply:
If you look at ln 44, I think the question is required?

Reply:
Just tested out adding a `required: true` in line with the yes/no labels, but that just made the individual checkbox required, instead of making the question required.